docker-run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Shell script invoked by next-build docker container (see docker/Dockerfile).
+# Env vars must be present to populate cmd params.
+# Secrets (remaining required params) must be present at ./envs/.env.${ENV}.
+
+APP_ENV=${ENV} yarn dev \
```

Comment:
Seeing as these are for the BRD-adjacent preview servers, I think ultimately we will want to be running `yarn build:preview` && `yarn start` instead of `yarn dev`, but this is fine for now